splunk.json

```diff
@@ -990,7 +973,27 @@
                     "description": "Validate event_ids",
                     "data_type": "boolean",
                     "default": false,
+                    "order": 8
```

Comment:
Maybe this could be moved below so we go 6-7-8 and not 8-6-7?

Reply:
done